### ISSUE TITLE
Request diagnostic plots for multiple regression

### ISSUE THREAD

**Issue by gulgit** [JuliaPlots/StatPlots.jl#122]
Request diagnostic plots for multiple regression. Some thing like this will be helpful
https://www.andrew.cmu.edu/user/achoulde/94842/homework/regression_diagnostics.html

**Comment by mkborregaard** [comment]
It'd be nice if you could give a little more info on exactly which of these you'd like and why.

**Comment by mkborregaard** [comment]
These are essentially just R's standard diagnostic plots when plotting a model, aren't they?

**Comment by gulgit** [comment]
Yes. Something in R like plot(model) with these following graphs would be very hand

1. Residuals vs Fitted
2. Q-Q plot of standardised residuals
3. Square root of residuals vs Fitted (Scale location)
4. Cook's distance

Reference link : http://www.theanalysisfactor.com/linear-models-r-diagnosing-regression-model/
